autoware_launch/config/control/trajectory_follower/lateral/mpc.param.yaml

```diff
@@ -75,4 +75,5 @@
       average_num: 1000
       steering_offset_limit: 0.02
 
-    debug_publish_predicted_trajectory: true  # publish debug predicted trajectory in Frenet coordinate
+    debug_publish_predicted_trajectory: true  # publish predicted trajectory for debugging, considering Frenet coordinate system and delayed vehicle state
```

Comment:
We do not expect that the predicted trajectory in the world coordinate will be published with this comment.